src/sonic-yang-models/yang-models/sonic-copp.yang

```diff
@@ -163,6 +163,11 @@ module sonic-copp {
 					}
 					description "reference to CoPP group";
 				}
+
+				leaf always_enabled {
+					type string;
```

Comment:
There is a type defined for true or false called boolean. Can you please set it https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-bgp-global.yang#L101